### PR TITLE
fix top bar and shadow

### PR DESCRIPTION
**Pull request by Ambradibi**
Fix top bar and shadow

Schermata 2019-04-24 alle 11 01 41
Schermata 2019-04-24 alle 11 02 18
Schermata 2019-04-24 alle 11 02 26

### CODE REVIEW

File: aegis-web/yo/app/views/login.html [hunk 1]
```diff
@@ -88,7 +88,7 @@
 
           </form>
           <div class="pull-right">
-            <a class="text-muted" href="/hopsworks-admin/security/login_issue.xhtml">Login help?</a>
+            <a class="text-muted" href="../../../../hopsworks-admin/src/main/webapp/security/login_issue.xhtml">Login help?</a>
```
Comment (cbotsikas) [inline aegis-web/yo/app/views/login.html]: Remove or revert this change

File: aegis-web/yo/app/views/datasetsBrowser.html [hunk 2]
```diff
@@ -52,18 +51,61 @@
...
+          <div class="row content-utility-bar content-utility-bar-actions" ng-if="!mainCtrl.showSearchPage">
               <div class="col-md-3">
                 <h4 style="margin-top: 10px; margin-bottom: 5px"><a ng-click="datasetsCtrl.goToDataSetsDir()">DataSets</a>
                 <span ng-show="datasetsCtrl.pathArray.length > 10">...</span>
                 <span data-ng-repeat="path in datasetsCtrl.pathArray.slice(datasetsCtrl.pathArray.length - 1 - datasetsCtrl.breadcrumbLen()) track by $index">
-                  <span> > </span><a ng-click="datasetsCtrl.goToFolder($index + 1)">{{path| limitTo: 16 }}{{path.length > 16 ? '..' : ' '}}</a>
+                  <span> > </span><span ng-click="datasetsCtrl.goToFolder($index + 1)">{{path| limitTo: 16 }}{{path.length > 16 ? '..' : ' '}}</span>
                 </span>
-                <span><a ng-click="datasetsCtrl.goToFolder(datasetsCtrl.pathArray.length)"> <span class="fa fa-refresh"></span></a></span>
+                <span><a ng-click="datasetsCtrl.goToFolder(datasetsCtrl.pathArray.length)"></a></span>
```
Comment (cbotsikas) [inline aegis-web/yo/app/views/datasetsBrowser.html]: Is there is an option to refresh now? If not, I would add a button with the rest (Go Up, New Folder etc)

File: aegis-web/yo/app/views/datasets.html [hunk 3]
```diff
@@ -132,7 +132,7 @@ <h4 style="margin-top: 10px; margin-bottom: 5px">
                 </div>
               </div>
             </div>
-            <div class="row row-no-margin-left" ng-controller="SearchResultCtrl as searchResultCtrl">
+            <div class="row" ng-controller="SearchResultCtrl as searchResultCtrl">
               <div class="ds-content col-lg-12" ng-class="{'sidenav-open': datasetsCtrl.tgState == true}" style="overflow: hidden;">
                 <div class="row" resizable r-directions="['bottom']" ng-class="{'rg-row-top-ds': readme !== null}">
```
Comment (cbotsikas) [inline aegis-web/yo/app/views/datasets.html:137]: The problem with the shadow wasn't that it was shown a bit on the left. 
It's ok that you've removed the margin, but the main problem is the white background. 
Remove the white background from `rg-row-top-ds` class from [angular-resizable.css#L142](https://github.com/aegisbigdata/hopsworks/blob/dev/aegis-web/yo/app/styles/angular-resizable.css#L142)

**Review by cbotsikas — CHANGES_REQUESTED** [review]
Have a look on my inline comments. 
I would also move the buttons to the white action bar, where the fileRef box is.
You could either make the fileRef smaller so that all 4 buttons can fit together with the box or maybe move the fileRef box to the Details pane on the right.

File: aegis-web/yo/app/views/login.html (hunk 1 above)
Comment (Ambradibi) [reply aegis-web/yo/app/views/login.html]: it was a test, my mistake! done

File: aegis-web/yo/app/views/datasetsBrowser.html (hunk 2 above)
Comment (Ambradibi) [reply aegis-web/yo/app/views/datasetsBrowser.html]: if you want I put the refresh button that was there before

File: aegis-web/yo/app/views/datasets.html (hunk 3 above)
Comment (Ambradibi) [reply aegis-web/yo/app/views/datasets.html:137]: done

File: aegis-web/yo/app/views/datasetsBrowser.html [hunk 4]
```diff
@@ -52,18 +51,61 @@
...
+                </ul>
+              </div>
+            </div>
+          <div class="row content-utility-bar content-utility-bar-actions" ng-if="!mainCtrl.showSearchPage">
               <div class="col-md-3">
                 <h4 style="margin-top: 10px; margin-bottom: 5px"><a ng-click="datasetsCtrl.goToDataSetsDir()">DataSets</a>
                 <span ng-show="datasetsCtrl.pathArray.length > 10">...</span>
                 <span data-ng-repeat="path in datasetsCtrl.pathArray.slice(datasetsCtrl.pathArray.length - 1 - datasetsCtrl.breadcrumbLen()) track by $index">
-                  <span> > </span><a ng-click="datasetsCtrl.goToFolder($index + 1)">{{path| limitTo: 16 }}{{path.length > 16 ? '..' : ' '}}</a>
+                  <span> > </span><span ng-click="datasetsCtrl.goToFolder($index + 1)">{{path| limitTo: 16 }}{{path.length > 16 ? '..' : ' '}}</span>
```
Comment (cbotsikas) [inline aegis-web/yo/app/views/datasetsBrowser.html]: Also, i just noticed this change. You shouldn't camouflage clickable links to text. Keep the a tag.

File: aegis-web/yo/app/views/datasetsBrowser.html (hunk 2 above)
Comment (cbotsikas) [reply aegis-web/yo/app/views/datasetsBrowser.html]: Since this is essentially doing the same as clicking the last path above, I would try to put ` <span class="fa fa-refresh fa-sm"></span>` in the same a tag as the text with an nf-if statement so that it's visible only on the last one. `fa-sm` class should make the icon smaller to match the text size. In any case, remove this line.

File: aegis-web/yo/app/views/datasetsBrowser.html (hunk 4 above)
Comment (cbotsikas) [reply aegis-web/yo/app/views/datasetsBrowser.html]: Are you changing this?

Comment (Ambradibi) [reply aegis-web/yo/app/views/datasetsBrowser.html]: i kept <a ng-click="datasetsCtrl.goToFolder($index + 1)">{{path| limitTo: 16 }}{{path.length > 16 ? '..' : ' '}}</a>

Comment (cbotsikas) [reply aegis-web/yo/app/views/datasetsBrowser.html]: I was talking about the change from `a` to `span`. You should keep the `a` tag instead

Comment (Ambradibi) [reply aegis-web/yo/app/views/datasetsBrowser.html]: sorry I kept <a ng-click="datasetsCtrl.goToFolder($index + 1)">{{path| limitTo: 16 }}{{path.length > 16 ? '..' : ' '}}</a> like before

**Comment by Ambradibi** [comment]
I didn't understand 'nf-if statement so it's only visible on the last' and 'In any case, remove this line'

File: aegis-web/yo/app/views/datasetsBrowser.html (hunk 4 above)
Comment (cbotsikas) [reply aegis-web/yo/app/views/datasetsBrowser.html]: image
Revert this change

Comment (Ambradibi) [reply aegis-web/yo/app/views/datasetsBrowser.html]: I did, but you don't see it yet

Comment (cbotsikas) [reply aegis-web/yo/app/views/datasetsBrowser.html]: commit and push so that i can merge this PR

Comment (Ambradibi) [reply aegis-web/yo/app/views/datasetsBrowser.html]: Schermata 2019-04-24 alle 14 33 35

I don't know why it doesn't show the right message

**Comment by cbotsikas** [comment]
> I didn't understand 'nf-if statement so it's only visible on the last' and 'In any case, remove this line'

Nevermind

File: aegis-web/yo/app/views/datasetsBrowser.html (hunk 4 above)
Comment (cbotsikas) [reply aegis-web/yo/app/views/datasetsBrowser.html]: you need to wrap your text with backtick ( ` ) when writing code in github comments. If it's a code block, you need triple backtick ( ``` )
Read more:

- https://daringfireball.net/projects/markdown/syntax#code
- https://help.github.com/en/articles/creating-and-highlighting-code-blocks

Comment (Ambradibi) [reply aegis-web/yo/app/views/datasetsBrowser.html]: thanks

**Review by cbotsikas — APPROVED** [review]
